frontend/src/components/ListDetail.js

```diff
@@ -2,27 +2,27 @@ import { useEffect, useState } from 'react';
 import { addItem, setCompletion, deleteItem } from "../http/lists";
 import { Item } from './Item';
 
-export const ListDetail = ({ id, name, completed, items, handleUpdate, removeList }) => {
-    const [messageState, setMessageState] = useState('');
-    const [errorMessageState, setErrorMessageState] = useState('');
+export const ListDetail = ({ id, name, completed, items, handleUpdate, removeList, setReturnError, setMessageState }) => {
     const [itemsState, setItemsState] = useState(items ? items : []);
     const [newItemName, setNewItemName] = useState('');
     const [currentName, setCurrentName] = useState(name)
+    const [errorMessage, setErrorMessage] = useState(null)
```

Comment:
Why is the default state for errorMessage, setErrorMessage a null but for the other string (newitemName) it's an empty string?

Reply:
addressed